docs/digger/admin/about-admin-guide.asciidoc

```diff
@@ -1,8 +1,5 @@
 == AeroGear Digger Administration
+The following text describes optional post-installation steps:
 
-The following text serves as a reference for AeroGear Digger components administration:
-
-- Nagios monitoring
-- Installing and configuring Jenkins Plugins
-- Using HTTP proxy for Digger installation and ...
-- 
\ No newline at end of file
+- Installing Jenkins Plugins
```

Comment:
@pmacko1 why did the sections change here?

Reply:
@laurafitzgerald to reflect the changes to the guide
these mr's are just splinters from RHMAP-18330, checkout that branch to see how the whole repo looks with all changes

Reply:
It is just a list of components that can be "administrated" after installation. Actually I was thinking about removing the administration guide altogether

Reply:
@pmacko1 what is the motivation behind removing the http-proxy one? Is the section about it also removed from the admin guide? removing the guide would be a separate JIRA.

Reply:
@laurafitzgerald use-http-proxy was moved to installation guide.

Reply:
@pmacko1 okay thanks for the clarification. feel free to merge this one.